Automattic/vip-go-mu-plugins
# Bump Cron Control

Fix errors for `wp orchestrate site list` (https://github.com/Automattic/Cron-Control/pull/193)

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

On a multisite install run `wp cron-control orchestrate site list`. Should return a list of correct URLs.
